src/app/button-confirm/button-confirm.component.spec.ts

```diff
@@ -0,0 +1,24 @@
+import {async, ComponentFixture, TestBed} from '@angular/core/testing';
+
+import {ButtonConfirmComponent} from './button-confirm.component';
+
+describe('ButtonConfirmComponent', () => {
+  let component: ButtonConfirmComponent;
+  let fixture: ComponentFixture<ButtonConfirmComponent>;
+
+  beforeEach(async(() => {
+    TestBed.configureTestingModule({
+      declarations: [ButtonConfirmComponent],
+    }).compileComponents();
+  }));
+
+  beforeEach(() => {
+    fixture = TestBed.createComponent(ButtonConfirmComponent);
+    component = fixture.componentInstance;
+    fixture.detectChanges();
+  });
+
+  it('should create', () => {
```

Comment:
la it clause dovrebbe essere un pelo più discorsiva, in modo che nella pagina dei test result venga fuori "quasi" una frase:

![Capture](https://user-images.githubusercontent.com/30255227/77851002-a72e7080-71d6-11ea-95d9-42d59c41f723.JPG)

tipo, secondo me li ci starebbe un "should create the button", "should create the login form"

Reply:
> a me title vien fuori così:

intendi {{title}}?
Se sì, colpa mia, sto facendo delle prove per modificare il titolo della pagina ogni volta che "cambiamo" pagina, anche se il cambiamento non avviene realmente. Se lasciamo il title _Stalker - GruppOne_ che avevi impostato tu in index.html quel titolo non cambierà mai, quindi stavo lavorando a sta cosa

Reply:
ok. ho una mezza idea che il problema sia negli spazi: `{{ title }}` e non `{{title}}`. comunque ho aggiunto una estensione a vscode che si chiama "angular" che dovrebbe semplificare molto la cosa (fa gli autocomplete delle direttive)

Reply:
no il problema è che quella sintassi funziona se il file html fa da template a un component, però il titolo della pagina viene definito nel file index.html che non è legato a nessun componente
https://angular.io/guide/set-document-title
questo risolve il problema con un servizio di angular, adesso lo implemento

Reply:
magari approfittane per ristrutturare i servizi in sottocartelle